core/src/test/java/org/infinispan/distribution/rehash/NonTxPrimaryOwnerBecomingNonOwnerTest.java

```diff
@@ -124,14 +124,9 @@ private void doTest(final TestWriteOperation op) throws Exception {
       checkPoint.trigger("allow_topology_" + duringJoinTopologyId + "_on_" + address(2));
 
       // Wait for the write CH to contain the joiner everywhere
-      eventually(new Condition() {
-         @Override
-         public boolean isSatisfied() throws Exception {
-            return cache0.getRpcManager().getMembers().size() == 3 &&
-                  cache1.getRpcManager().getMembers().size() == 3 &&
-                  cache2.getRpcManager().getMembers().size() == 3;
-         }
-      });
+      eventually(() -> cache0.getRpcManager().getMembers().size() == 3 &&
+            cache1.getRpcManager().getMembers().size() == 3 &&
+            cache2.getRpcManager().getMembers().size() == 3);
```

Comment:
Could be replaced with 3 `eventuallyEquals(3, () -> cacheX.getRpcManager().getMembers().size())` calls.